src/statCollector.ts

```diff
@@ -247,16 +247,16 @@ async function getNetworkStats(): Promise<ProcessedNetworkStats> {
   if (logger.isDebugEnabled()) {
     logger.debug(`Got network stats: ${JSON.stringify(response.data)}`)
   }
-
+  
   response.data.forEach((element: NetworkStats) => {
     networkReadX.push({
       x: element.time,
-      y: element.rxMb
-    })
+      y: element.rxMb || 0
```

Comment:
Can you do the same for y values of CPU and memory metrics?

Reply:
done.